packager/packager.cc

```diff
@@ -221,12 +371,21 @@ class PackageInfoImpl final : public core::packages::PackageInfo {
     core::Loc declLoc_;
     // The possible path prefixes associated with files in the package, including path separator at end.
     vector<string> packagePathPrefixes = {};
-    // The names of each package imported by this package.
-    vector<Import> importedPackageNames = {};
+
     // List of exported items that form the body of this package's public API.
     // These are copied into every package that imports this package.
     vector<Export> exports_ = {};
 
+    const Imports imports() const {
+        // Imports holds a non-const reference, but exposes a const interface.
+        return Imports{const_cast<vector<Import> &>(this->importedPackageNames)};
```

Comment:
What this comment is trying to say is that the `const Imports` prevents calling the mutable `Imports::add` method, right? That the `const_cast` is fine, because we return a `const Imports`

I think that makes sense to me. Not sure if the comment could have been clearer about that.